lib/server.js

```diff
@@ -31,16 +32,15 @@ server.start = () =>
         server.isOn = true;
         resolve();
```

Comment:
I'm not entirely sure this will work as intended. Line 33 may need to become `resolve(server.http)` if you want to access `server.http` later on.

Reply:
Truthfully, I am not 100% certain on what this does as I used a code fellows project from my time there as a guide for building this out. I made the changes you suggested here and will do some do some digging to gain a better understanding of this.  Thanks!